src/components/__tests__/NotAuthorisedView.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import { fireEvent } from '@testing-library/react-native'
+import { mocked } from 'jest-mock'
+import React from 'react'
+import { Linking } from 'react-native'
+
+import { getLabels } from '../../services/helpers'
+import render from '../../testing/render'
+import NotAuthorisedView from '../NotAuthorisedView'
+
+jest.mock('react-native/Libraries/Linking/Linking', () => ({
+  openSettings: jest.fn(),
+}))
+
+describe('NotAuthorisedView', () => {
+  const setVisible = jest.fn()
+  it('should render description', () => {
+    const { getByTestId } = render(
+      <NotAuthorisedView setVisible={setVisible} description={getLabels().general.camera.noAuthorization.description} />
+    )
+    expect(getByTestId('no-auth')).toBeTruthy()
```

Comment:
This isn't actually checking for the description though, it's checking for the container. You should be able to get the description with `getByText(getLabels().general.camera.noAuthorization.description)` (for which you might want to set a well-named variable).

Reply:
I was not sure about it as well. I will fix it, thanks